ui/apps/platform/src/Containers/Policies/Wizard/Step4/PolicyScopeForm.tsx

```diff
@@ -87,7 +87,7 @@ function PolicyScopeForm() {
         // Move request to exclusion card to add restSearch for cluster or namespace if specified in exclusion scope.
         // Search element to support creatable deployment names.
         const restSort = { field: 'Deployment' }; // ascending by name
-        fetchDeployments([], restSort, 0, 0)
+        fetchDeploymentsWithProcessInfo([], restSort, 0, 0)
```

Comment:
As a follow up, this might become `listDeployments` function, because only `id` and `name` is needed:

https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Policies/Wizard/Step4/PolicyScopeCard.tsx#L150-L157

Reply:
Thank you for reminding me about that. Policies does not use that option.